README.md

```diff
@@ -38,3 +38,15 @@ Then, you can use rosrun to run the simulator
 ```shell
 rosrun carla_binary_catkin CarlaUE4
 ```
+
+## Next Steps 
+
+If you are interested in running the CARLA server using a configuation (.ini) file, or in connecting to the server using a Python client, you can find the contents of the extracted CARLA binary at the following location in your catkin workspace:
```

Comment:
It's not the binary that is extracted but the `.tar.gz` or `.zip` file, right?

Reply:
Yup, that's correct - I'll update the README accordingly.